src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -0,0 +1,380 @@
+                    }
+
+                    var tensors = runner.Fetch(_outputColName).Run();
+
+                    Contracts.Assert(tensors.Length > 0);
+
+                    var values = dst.Values;
+                    if (Utils.Size(values) < _outputColType.VectorSize)
+                        values = new T[_outputColType.VectorSize];
+
+                    TensorFlowUtils.FetchData<T>(tensors[0].Data, values);
+                    dst = new VBuffer<T>(values.Length, values, dst.Indices);
+                };
+                return valuegetter;
+            }
+
+            public Delegate[] CreateGetters(IRow input, Func<int, bool> activeOutput, out Action disposer)
+            {
+                var getters = new Delegate[1];
+                disposer = null;
+                using (var ch = _host.Start("CreateGetters"))
+                {
+                    if (activeOutput(0))
+                        getters[0] = MakeGetter(input);
+                    ch.Done();
+                    return getters;
+                }
+            }
+
+            public Func<int, bool> GetDependencies(Func<int, bool> activeOutput)
+            {
+                return col => activeOutput(0) && _inputColIndices.Any(i => i == col);
+            }
+
+            public RowMapperColumnInfo[] GetOutputColumns()
+            {
+                var info = new RowMapperColumnInfo[1];
+                info[0] = new RowMapperColumnInfo(_outputColName, _outputColType, null);
+                return info;
+            }
+
+            private static (ColumnType, TFDataType) GetOutputTypes(TFGraph graph, string columnName)
+            {
+                var tfoutput = new TFOutput(graph[columnName]);
+                var shape = graph.GetTensorShape(tfoutput);
+
+                int[] dims = new int[shape.NumDimensions];
+                for (int k = 0; k < shape.NumDimensions; k++)
+                    dims[k] = (int)(shape[k] == -1 ? 1 : shape[k]);
+
+                var type = TensorFlowUtils.Tf2MlNetType(tfoutput.OutputType);
+                return (new VectorType(type, dims), tfoutput.OutputType);
+            }
+
+            private static (string[], int[], bool[], TFShape[], TFDataType[]) GetInputMetaData(TFGraph graph, string[] source, ISchema inputSchema)
+            {
+                var tfShapes = new TFShape[source.Length];
+                var tfTypes = new TFDataType[source.Length];
+                var colNames = new string[source.Length];
+                var inputColIndices = new int[source.Length];
+                var isInputVector = new bool[source.Length];
+                for (int i = 0; i < source.Length; i++)
+                {
+                    colNames[i] = source[i];
+                    if (!inputSchema.TryGetColumnIndex(colNames[i], out inputColIndices[i]))
+                        throw Contracts.Except($"Column '{colNames[i]}' does not exist");
+
+                    var tfoutput = new TFOutput(graph[colNames[i]]);
```

Comment:
>graph[colNames[i]] [](start = 48, length = 18)

Also need to check if any op with name `colNames[i]` exist in TF graph otherwise it will throw general dictionary key not found exception here.